tfhe/docs/getting_started/operations.md

```diff
@@ -1,64 +1,484 @@
+* [PartialEq](https://doc.rust-lang.org/std/cmp/trait.PartialEq.html)
+
+A simple example on how to use these operations:
+
+```rust
+use tfhe::prelude::*;
+use tfhe::{generate_keys, set_server_key, ConfigBuilder, FheUint3};
+
+fn main() -> Result<(), Box<dyn std::error::Error>> {
+    let config = ConfigBuilder::all_disabled().enable_default_uint3().build();
+    let (keys, server_keys) = generate_keys(config);
+    set_server_key(server_keys);
+    
+    let clear_a = 7;
+    let clear_b = 3;
+    
+    let mut a = FheUint3::try_encrypt(clear_a, &keys)?;
+    let mut b = FheUint3::try_encrypt(clear_b, &keys)?;
+    
+    assert_eq!(a.gt(&b).decrypt(&keys) != 0, true);
+    assert_eq!(b.le(&a).decrypt(&keys) != 0, true);
+
+    Ok(())
+}
+```
+
+### Univariate function evaluations.
+
+The shortint type also supports the computation of univariate functions, which deep down uses TFHE's _programmable bootstrapping_.
+
+A simple example on how to use these operations:
+
+```rust
+use tfhe::prelude::*;
+use tfhe::{generate_keys, set_server_key, ConfigBuilder, FheUint4};
+
+fn main() -> Result<(), Box<dyn std::error::Error>> {
+    let config = ConfigBuilder::all_disabled().enable_default_uint4().build();
+    let (keys, server_keys) = generate_keys(config);
+    set_server_key(server_keys);
+
+    let pow_5 = |value: u64| {
+        value.pow(5) % FheUint4::MODULUS as u64
+    };
+
+    let clear_a = 12;
+    let a = FheUint4::try_encrypt(12, &keys)?;
+
+    let c = a.map(pow_5);
+    let decrypted = c.decrypt(&keys);
+    assert_eq!(decrypted, pow_5(clear_a) as u8);
+
+    Ok(())
+}
+```
+
+### Bivariate function evaluations.
+
+Using the shortint type allows you to evaluate bivariate functions (i.e., functions that takes two ciphertexts as input).
+
+A simple code example:
+
+```rust
+use tfhe::prelude::*;
+use tfhe::{generate_keys, set_server_key, ConfigBuilder, FheUint2};
+
+fn main() -> Result<(), Box<dyn std::error::Error>> {
+    let config = ConfigBuilder::all_disabled().enable_default_uint2().build();
+    let (keys, server_keys) = generate_keys(config);
+    set_server_key(server_keys);
+    
+    let clear_a = 1;
+    let clear_b = 3;
+    let a = FheUint2::try_encrypt(clear_a, &keys)?;
+    let b = FheUint2::try_encrypt(clear_b, &keys)?;
+
+    
+    let c = a.bivariate_function(&b, std::cmp::max);
+    let decrypted = c.decrypt(&keys);
+    assert_eq!(decrypted, std::cmp::max(clear_a, clear_b) as u8);
+
+    Ok(())
+}
+```
 
 ## Integer
 
-In TFHE-rs, integers represent unsigned integers up to 256 bits. They are encoded using Radix representations by default (more details [here](../integer/operations.md)).
+In TFHE-rs, integers are used to encrypt any messages larger than 4 bits. All supported operations are listed below.
+
+### Arithmetic operations.
+
+Homomorphic integer types support arithmetic operations.
+
+The list of supported operations is:
+
+| name                                                    | symbol | type   |
```

Comment:
does it make sense to repeat those as we have a table at the start of the file ?